content/php/concepts/math-functions/terms/abs/max/max.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: 'Max()'
+Description: 'Used to find the highest value in an array or a list of variables. It returns the highest value from the input provided.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+ - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`max()`** function in PHP is used to find the highest value in an [array](https://www.codecademy.com/resources/docs/php/arrays) or a list of [variables](https://www.codecademy.com/resources/docs/php/variables). It returns the highest value from the input provided.
+
+## Syntax
+
+```pseudo
+max($var1, $var2, $var3, ...);
+```
+
+`$var1`, `$var2`, `$var3` are the [variables](https://www.codecademy.com/resources/docs/php/variables) or [array](https://www.codecademy.com/resources/docs/php/arrays) from which the maximum value will be returned.
```

Comment:
Need to mention the difference between calling this with a list of variables (multiple parameters) and calling it with an array (single parameter)